src/mozillavpn.cpp

```diff
@@ -1165,11 +1165,6 @@ void MozillaVPN::controllerStateChanged() {
   NetworkManager::instance()->clearCache();
 }
 
-void MozillaVPN::backendServiceRestore() {
-  logger.debug() << "Background service restore request";
-  // TODO
-}
```

Comment:
Thank you for removing this. We need to notify QA and the team about this change somehow. Maybe we can create a separate 0 pointer for it just for tracking purposes?